src/main/config.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <sstream>
+#include "gflags/gflags.h"
+#include "config.hpp"
+
+extern "C" {
+	#include "exec_monitor.h"
+}
+
+/*
+	FLAGS SECTION
+*/
+
+// EXEC FLAG - MONITOR PROCESS EXECUTION
+DEFINE_bool(exec, false, "Monitors process executions.");
```

Comment:
I thought `gflags` had been largely replaced by Abseil flags? (https://abseil.io/docs/cpp/guides/flags)
Is this to minimize dependencies? (Both are fine though.)

Reply:
Hi! Kind of, yes, because gflags only deals with... flags, whereas abseil has so many other utilities.